script.js

```diff
@@ -0,0 +1,7 @@
+var alertButton = document.getElementsByClassName('button');
+
+for (var x=0; x<alertButton.length; x++) {
```

Comment:
checking length of array move above loop to new variable, because in that case after every single loop rotation lenght is calculating again